src/message.rs

```diff
@@ -173,7 +173,7 @@ pub struct MessageKind {
     pub content: Option<Content>,
 }
 
-type Amount = u32;
+type Amount = i64;
```

Comment:
why to allow negative amounts on fiat currency?

Reply:
You are right, but i just did this because in order struct we defined fiat_amount field as i64 so it was easier to manage with this type, but i agree that we should change.